syntax = "proto3";

package sensor_data;

service SensorDataService {
    rpc SendPressureReading (SensorData) returns (Empty);
}

message SensorData {
    string sensor_id = 1;
    int32 person_id =2;
    float systolicPressure = 3;
    float diastolicPressure = 4;
    string unit = 5;
    string date = 6;
}

message Empty {}